tests/src/main/java/org/hibernate/beanvalidation/tck/tests/methodvalidation/ValidateParametersTest.java

```diff
@@ -236,22 +226,17 @@ public void testCrossParameterConstraintGivenSeveralTimes() throws Exception {
 				parameterValues
 		);
 
-		assertCorrectNumberOfViolations( violations, 2 );
+		assertNumberOfViolations( violations, 2 );
 
 		assertCorrectConstraintTypes(
 				violations,
 				MyCrossParameterConstraint.class,
 				MyCrossParameterConstraint.class
 		);
-		assertCorrectPathNodeNames(
-				violations,
-				names( methodName, TestUtil.CROSS_PARAMETER_NODE_NAME ),
-				names( methodName, TestUtil.CROSS_PARAMETER_NODE_NAME )
-		);
-		assertCorrectPathNodeKinds(
-				violations,
-				kinds( ElementKind.METHOD, ElementKind.CROSS_PARAMETER ),
-				kinds( ElementKind.METHOD, ElementKind.CROSS_PARAMETER )
+		assertThat( violations ).containsOnlyPaths(
```

Comment:
I suspect we should use http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractIterableAssert.html#containsExactlyInAnyOrder(ELEMENT...) instead of containsOnly. I'm pretty sure it's a new thing in assertj and it wasn't existing when it was introduced.

I think it would be a nice change but for another PR :). Opening an issue for that.